### ISSUE TITLE
Coach view: ability to add independent practice quizzes to a lesson

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#8514]
### User stories for this issue:

> As a coach, I want to see independent practice quizzes listed and be able to preview one that is selected, so that I can add them into a lesson. 

|Frame|Explanation|
|--|--|
|![Screen Shot 2021-10-21 at 11 38 35 AM](https://user-images.githubusercontent.com/13563002/138338320-dcd2349c-c813-445a-8990-7440ec89cb27.png)|Independent practice quizzes appear in channel content as card|
|![Screen Shot 2021-10-21 at 11 38 47 AM](https://user-images.githubusercontent.com/13563002/138337750-ff2580a5-7385-4edc-814a-0bcaefc01de0.png)| Preview while browsing has "Add" button to add to lesson resources |
|![Screen Shot 2021-10-21 at 11 40 24 AM](https://user-images.githubusercontent.com/13563002/138337760-1a42adc8-fe77-42ad-b813-0eb8b35a4f5d.png)| Appearance after coach returns to Plan > Lesson, note: new independent practice quiz is present |
|![Screen Shot 2021-10-21 at 11 40 13 AM](https://user-images.githubusercontent.com/13563002/138337756-883183a7-473c-4474-bf0e-9fce5a788499.png)| If coach clicks for preview, note that the top bar is now changed to go back to the Lesson and the "Add" button is no longer there| 

### Notes and changes to current designs:
- [Figma for adding practice quiz to lesson](https://www.figma.com/file/TfaSTT9gs6fPKhGdNUjffU/Channel-quizzes?node-id=576%3A648)
- Routing and navigation has already been created for this user story
- All strings required should already be in `commonCoreStrings`, `commonCoachStrings`, or `commonLearnStrings`

### Criteria for success:
**Please copy these into your PR for "Steps for Testing"!**

_This issue is done when the checkbox and Gherkin story below works_

- [ ] coaches can add an independent practice quiz to a lesson resource list
```
Feature: Independent practice quizzes can be added to lessons
	Scenario: coaches can add an independent practice quiz to a lesson resource list
		Given that I am on the lesson resource management page
		When I browse the content tree
			And I find and select a checkbox to the left of the independent practice quiz card
		Then I see a snackbar appear confirming my action
		When I exit the lesson resource management page
		Then I see that the independent practice quiz is in the lesson resource list
```
- [ ] coaches can preview an independent practice quiz before adding it to a lesson
```
Feature: Independent practice quizzes can be added to lessons
	Scenario: coaches preview an independent practice quiz before adding it to a lesson
		Given that I am on the lesson resource management page
		When I browse the content tree
			And I find and click an independent practice quiz content card
		Then I see a preview of the independent practice quiz

```

**Comment by bjester** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Kolibri-Ecosystem-6047e525b8b9d900182e2d6f/issues/learningequality/kolibri/8514?planning-poker) with ZenHub @jredrejo @marcellamaki @sairina @rtibbles @MisRob @nucleogenesis @jamalex

**Comment by rtibbles** [comment]
Fixed in #8544
